mdp/nodes/sfa_nodes.py

```diff
@@ -369,6 +371,66 @@ def get_eta_values(self, t=1):
         return self._refcast(t / (2 * numx.pi) * numx.sqrt(self.d))
 
 
+class UnevenlySampledSFANode(SFANode):
+
+    def __init__(self, input_dim=None, output_dim=None, dtype=None,
+                 include_last_sample=True, rank_deficit_method='none'):
+        super(SFANode, self).__init__(input_dim, output_dim, dtype)
```

Comment:
Thanks for the great suggestion!

Should `super(UnevenlySampledSFANode, self).__init__(input_dim, output_dim, dtype, include_last_sample, rank_deficit_method)`
not be sufficient?

Reply:
Yes I think so. Maybe we wouldn't even need an `__init__` in `UnevenlySampledSFANode` at all, but I am not sure right now how Python's OOP would handle that. There are to behaviours I can think of:

* it wouldn't call `__init__` at all
* it would call the the first `__init__` it can find int the mro, i.e. in `__mro__`.

The second case would mean that it would behave like a plain `super(UnevenlySampledSFANode, self).__init__(input_dim, output_dim, dtype, include_last_sample, rank_deficit_method)`-call if we omit `__init__` in `UnevenlySampledSFANode`. If that's the case we should actually omit it.